odoo/addons/base/models/ir_fields.py

```diff
@@ -17,12 +17,12 @@ def exclude_ref_fields(record):
     return {k: v for k, v in record.items() if k not in REFERENCING_FIELDS}
 
 CREATE = lambda values: (0, False, values)
-UPDATE = lambda id, values: (1, id, values)
-DELETE = lambda id: (2, id, False)
-FORGET = lambda id: (3, id, False)
-LINK_TO = lambda id: (4, id, False)
-DELETE_ALL = lambda: (5, False, False)
-REPLACE_WITH = lambda ids: (6, False, ids)
+UPDATE = lambda id, values: (fields.X2ManyCmd.UPDATE, id, values)
+DELETE = lambda id: (fields.X2ManyCmd.DELETE, id, False)
+FORGET = lambda id: (fields.X2ManyCmd.UNLINK, id, False)
+LINK_TO = lambda id: (fields.X2ManyCmd.LINK, id, False)
+DELETE_ALL = lambda: (fields.X2ManyCmd.CLEAR, False, False)
+REPLACE_WITH = lambda ids: (fields.X2ManyCmd.SET, False, ids)
```

Comment:
should rewrite the file to use the command function defined on `fields.X2ManyCmd`